src/c++/perf_analyzer/genai-perf/tests/test_cli.py

```diff
@@ -178,6 +178,7 @@ def test_help_version_arguments_output_and_exit(
         ],
     )
     def test_non_file_flags_parsed(self, monkeypatch, arg, expected_attributes, capsys):
+        logging.init_logging()
```

Comment:
Does this work?
The logger needs an entry in the logger list in logger.py.

Reply:
It worked when I added this in this [PR](https://github.com/triton-inference-server/client/pull/645/files). I think it works because the `parser.py` is in the logger list so all the loggings happening in it is being captured.

Reply:
makes sense. thanks for explaining that to me